modules/rgbd/include/opencv2/rgbd/large_kinfu.hpp

```diff
@@ -17,6 +17,72 @@ namespace cv
 {
 namespace large_kinfu
 {
+struct CV_EXPORTS_W VolumeParams
+{
+    enum VolumeKind
+    {
+        TSDF = 0,
+        HASHTSDF = 1,
+        COLOREDTSDF = 2
+    };
+
+    /** @brief Kind of Volume
+        Values can be TSDF (single volume) or HASHTSDF (hashtable of volume units)
+    */
+    CV_PROP_RW int kind;
+
+    /** @brief Resolution of voxel space
+        Number of voxels in each dimension.
+        Applicable only for TSDF Volume.
+        HashTSDF volume only supports equal resolution in all three dimensions
+    */
+    CV_PROP_RW int resolutionX;
+    CV_PROP_RW int resolutionY;
+    CV_PROP_RW int resolutionZ;
+
+    /** @brief Resolution of volumeUnit in voxel space
+        Number of voxels in each dimension for volumeUnit
+        Applicable only for hashTSDF.
+    */
+    CV_PROP_RW int unitResolution = { 0 };
+
+    /** @brief Initial pose of the volume in meters, should be 4x4 float or double matrix */
+    CV_PROP_RW Mat pose;
+
+    /** @brief Length of voxels in meters */
+    CV_PROP_RW float voxelSize;
+
+    /** @brief TSDF truncation distance
+        Distances greater than value from surface will be truncated to 1.0
+    */
+    CV_PROP_RW float tsdfTruncDist;
+
+    /** @brief Max number of frames to integrate per voxel
+        Represents the max number of frames over which a running average
+        of the TSDF is calculated for a voxel
+    */
+    CV_PROP_RW int maxWeight;
+
+    /** @brief Threshold for depth truncation in meters
+        Truncates the depth greater than threshold to 0
+    */
+    CV_PROP_RW float depthTruncThreshold;
+
+    /** @brief Length of single raycast step
+        Describes the percentage of voxel length that is skipped per march
+    */
+    CV_PROP_RW float raycastStepFactor;
+
+    /** @brief Default set of parameters that provide higher quality reconstruction
+        at the cost of slow performance.
+    */
+    CV_WRAP static Ptr<VolumeParams> defaultParams(int _volumeType);
```

Comment:
again, underscore prefix in public header.

Reply:
done